spec/unit/matrixrtc/MatrixRTCSession.spec.ts

```diff
@@ -508,6 +508,20 @@ describe("MatrixRTCSession", () => {
 
                 jest.useFakeTimers();
 
+                // preparing the delayed disconnect should handle the delay being too long
+                const sendDelayedStateExceedAttempt = new Promise<void>((resolve) => {
+                    const error = new MatrixError({
+                        "errcode": "M_UNKNOWN",
+                        "org.matrix.msc4140.errcode": "M_MAX_DELAY_EXCEEDED",
+                        "org.matrix.msc4140.max_delay": 7500,
+                    });
```

Comment:
This doesn't seem to match the MSC https://github.com/matrix-org/matrix-spec-proposals/blob/toger5/expiring-events-keep-alive/proposals/4140-delayed-events-futures.md#m_invalid_param-instead-of-m_max_delay_exceeded